packages/react/src/components/Button/Button.mdx

```diff
@@ -26,39 +32,224 @@ import { Button } from '@digdir/design-system-react';
+<Canvas of={ButtonStories.Tertiary} />
+
+<br />
+
+## Fargevarianter
+
+### Hovedfarger
+
+Knappene kommer i to hovedfarger `first` og `second`. Hvis du har overstyrt fargepaletten vil fargene `first`og `second` være det du selv har satt.
+
+- `first`-fargen stikker seg mer ut enn `second` og brukes gjerne når handlingen skal ha oppmerksomhet.
+
+<Canvas of={ButtonStories.First} />
+
+- `second`-fargen er mer nøytral og kan brukes når det er flere knapper på en side som ikke skal ta så mye oppmerksomhet.
+
+<Canvas of={ButtonStories.Second} />
+
+### Danger
+
+`Danger`-fargen bruker vi til handlinger som brukeren ikke kan angre, for eksempel Slett.
+
+<Canvas of={ButtonStories.Danger} />
+
+### Success
+
+`Success`-fargen bruker vi bare når brukeren fullfører en prosess.
+
+<Canvas of={ButtonStories.Success} />
+
+### Kombinere ulike fargevarianter
+
+En gruppe med knapper bør bruke samme fargevariant. Unntaket er når vi skal tydeliggjøre en handling brukeren ikke kan angre, for eksempel Forkast. Vi skal ikke kombinere andre fargevarianter.
+
+<Canvas of={ButtonStories.KombinereFarger} />
 
-### Lenke
+<br />
 
-Sett `as="a"` dersom du trenger å bruke knappen som en lenke. Vurder først om det ikke holder å bruke [Link](/docs/felles-link--docs) komponenten.
+## Eksempler
+
+### Knapp som lenke
 
-<Canvas of={ButtonStories.UsedAsLink} />
+Bruk `asChild` dersom du trenger å bruke knappen som en lenke. Vurder først om det ikke holder å bruke [`Link`](/docs/felles-link--docs) komponenten.
+Du kan lese mer om komposisjon [her](https://www.designsystemet.no/grunnleggende/for-utviklere/komposisjon).
 
-### Knapp med tekst
+<Canvas of={ButtonStories.KnappSomLenke} />
 
-<Canvas of={ButtonStories.Variants} />
+### Knapp med kun ikon
 
-### Knapp med ikon
+Knapper med kun ikon bruker vi bare for godt kjente ikoner som Lukk og Slett. Når vi bruker knapper med kun ikon, må vi huske en tilgjengelig tekst som forklarer hva knappen gjør. Dette kan legges i `title`, `aria-label` eller i en `Tooltip` som i eksempelet under.
 
-<Canvas of={ButtonStories.WithIcon} />
+<Canvas of={ButtonStories.BareIkon} />
 
 ### Knapp med tekst og ikon
 
-<Canvas of={ButtonStories.WithIconAndText} />
+Når vi bruker ikon sammen med tekst, skal ikonet ha `aria-hidden` slik at teksten ikke leses opp av skjermleser to ganger.
+
+<Canvas of={ButtonStories.TekstOgIkon} />
+
+Vi plasserer ikonet til venstre eller høyre for teksten avhenger av hva knappen kommuniserer og hvor den plasseres i grensesnittet.
+
+Som regel er det **venstre** som er den beste plasseringen, men det finnes unntak:
+
+- Er det en knapp som sier “Start” med en høyrepekende pil, er det naturlig at pilen plasseres til høyre for teksten.
+- Er knappen fast plassert på høyre side av grensesnittet er det best at ikonet også er høyrestilt for teksten.
+
+<br />
 
 ### Full bredde
 
-<Canvas of={ButtonStories.FullWidth} />
+Noen ganger ser det best ut når knappen fyller hele bredden. For eksempel i bunnen av en liste.
+
```

Comment:
På eksempelet som er med her, skriv Last inn i stedet for bare Last

Reply:
Fikset ✅